docs/source/modules/k8s.models.configmap.rst

```diff
@@ -0,0 +1,7 @@
+k8s\.models\.configmap module
```

Comment:
These files seem to be autogenerated, should they perhaps not be checked in, and rather be generated when building docs?

Reply:
If readthedocs can generate it, then I think we should let it

Reply:
They are generated by `make html` at least, but I don't know what readthedocs uses.. Anyway, we can handle that in a different PR.